app.js

```diff
@@ -43,33 +53,77 @@ app.get('/resetpw', function (request, response) {
   response.render('resetpw');
 });
 
-// array of riders and rewards
-var metroRiders = data.rider;
-var metroRewards = data.reward;
-
-// display dashboard - with rider data
+// display dashboard - with rider data and rewards
 app.get('/dashboard', function (request, response) {
 	// pass data to template
-	response.render('dashboard', {
-		rider: metroRiders
-	});
+  var riders = db.get('riders').value()
+  var rewards = db.get('rewards').value()
+  response.render('dashboard', {riders: riders, rewards: rewards})
 });
 
-// display rewards page - reward options data
-app.get('/rewards', function (request, response) {
- // pass data to template
-	response.render('rewards', {
-		reward: metroRewards
-	});
+// create a new reward
+// app.post('/createReward', function(req, res) {
+  // get data from form
+  // var rewardType = req.body.rewardType;
+  // var pointsRequired = req.body.pointsRequired;
+
+  // insert new reward into database
+    // db.get('rewards')
+    //   .push({rewardType: rewardType, pointsRequired: pointsRequired, id: uuid()})
+    //   .write()
+
+  // redirect
+  //     res.redirect('redeem')
+  // })
+
+// Select reward from form, update balance
+// Redirect to redeem page to display redemption and new balance message
+  app.post('/redeem', function(request, response) {
+    // get data from form/dashboard page
+    // var rewardType = request.body.rewardType;
+    // var pointsRequired = request.body.pointsRequired;
+    // var id = request.params.id;
+    // var newBalance = pointBalance - pointsRequired;
+
+    //update user balance
+    db.get('riders')
+      .find({ name: 'Alice Green' })
+      .assign({ pointBalance: 550-20 })
+      .write()
+
+// redirect
+  response.redirect('redeem')
```

Comment:
Since you want the `redeem` page to show some data, you need to pass some data in the redirect. The only way to pass data in a redirect is through the url.

The two ways to pass data in an url is through parameters or query strings. I used parameters in the demos `(/route/:id)` but not query strings. 

If you want query strings, look for `req.query` on http://expressjs.com/en/api.html#req

 Assuming the reward id is 1, the  url can be `redeem/1` using parameters,  or  the url can be `redeem?id=1` using query strings. You need to adjust the `app.get('/redeem'` accordingly.